DANS-KNAW/easy-validate-dans-bag
# EASY-1701 Check dat een Is-Version-Of uuid ook daadwerkelijk bestaat

Fixes EASY-1701

#### When applied it will
* Check with a HTTP HEAD request that the UUID pointed to by `Is-Version-Of` in bag-info.txt resolves.
* Report a specific error if the bag store is off-line.

#### Where should the reviewer @DANS-KNAW/easy start?
[here](https://github.com/DANS-KNAW/easy-validate-dans-bag/compare/master...janvanmansum:EASY-1701a?expand=1#diff-932124b09a91424ddf46e43c7025441fR32)

#### How should this be manually tested?
* Create a bag that contains a non-resolving `Is-Version-Of` field and check that the violation is reported.
*  Stop the bag store service and check that the I/O Error violation is reported.
* Run the sequence of the revision01, revision02 and revision03 bags from `easy-sword-dans-examples` and verify that it goes through without violation reports.

#### Related to
https://github.com/DANS-KNAW/easy-dtap/pull/237
